src/Config.Binder/Resources.resx

```diff
@@ -132,4 +132,7 @@
   <data name="Error_UnsupportedMultidimensionalArray" xml:space="preserve">
     <value>Cannot create instance of type '{0}' because multidimensional arrays are not supported.</value>
   </data>
+  <data name="FormatError_FailedBindingRequiredProperty" xml:space="preserve">
+    <value>Required member '{0}' is missing or cannot be serialized.</value>
```

Comment:
cannot be _de_ serialized ?

Reply:
In this case I think the end of the message can be removed because serialization won’t trigger this error (old message)